app/constants/constants.py

```diff
@@ -8,3 +8,7 @@ class ListenerEventTypes(Enum):
     BEFORE_SERVER_START = "before_server_start"
     BEFORE_SERVER_STOP = "before_server_stop"
     AFTER_SERVER_STOP = "after_server_stop"
+
+
+class CorporateType(Enum):
+    corporate_type = ['employee', 'HR']
```

Comment:
Not a correct way to define enum. It should be like 
```
class CorporateType(Enum):
    EMPLOYEE =  'employee'
    HR = 'HR'
```

Also, valid corporate types are B2B2E, Insurance, API